internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -6159,6 +6159,18 @@ func TestHandleGetWork(t *testing.T) {
 		}(),
 		wantErr: true,
 		errCode: dcrjson.ErrRPCInternal.Code,
+	}, {
+		name:    "handleGetWork: no work during chain reorg",
+		handler: handleGetWork,
+		cmd:     &types.GetWorkCmd{},
```

Comment:
I believe you need to add `mockMiningState: mine(),` here in order to hit the code in question.

Reply:
Updated.  I wasn't able to use the using `mine` because that one doesn't set the previous state to the necessary value to skip the initial template retrieval via the template subscription, but you're correct that a mining state was needed.

I had originally put `mine` there when writing the test, but it was going down the aforementioned path incorrectly and I thought removing it would cause the default state to be set properly, but after digging in based on your comment here, I see that wasn't the case either.

Anyway, it's correct now.